src/ast/mod.rs

```diff
@@ -3302,22 +3307,27 @@ impl fmt::Display for Statement {
                 describe_alias,
                 verbose,
                 analyze,
+                query_plan,
                 statement,
                 format,
                 options,
             } => {
                 write!(f, "{describe_alias} ")?;
 
-                if *analyze {
-                    write!(f, "ANALYZE ")?;
-                }
+                if *query_plan {
+                    write!(f, "QUERY PLAN ")?;
+                } else {
+                    if *analyze {
+                        write!(f, "ANALYZE ")?;
+                    }
```

Comment:
can we skip the else condition if it works just as well to do?
```rust
if query_plan {
   write!()
}
if analyze {
  write!()
}
```
thinking that would keep this part simpler, even though the parser only accepts one or the other for the same statement

Reply:
@iffyio Yes, that makes sense. It's better to keep the fmt simpler.